src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -3021,4 +3021,9 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
         const _: i32 = 1;
         const _: i32 = 1;
     """)
+
+    fun `test no E0252 multiple underscore aliases`() = checkErrors("""
```

Comment:
Unfortunately, this test does nothing because all items in imports are from stdlib and cannot be resolved because you don't add stdlib dependency here
Try to add `@ProjectDescriptor(WithStdlibRustProjectDescriptor::class)` and it will fail.

Also, let's avoid using stdlib here not to slow down tests. You can declare inline module with some items and use them in imports